sda/host.go

```diff
@@ -612,3 +617,63 @@ func (h *Host) Rx() uint64 {
 func (h *Host) Address() string {
 	return h.workingAddress
 }
+
+//Received gives packets received
+func Received(n map[network.ServerIdentityID]network.SecureConn) uint64 {
+	var a uint64
+	for _, value := range n {
+		a = value.Rx()
+	}
+	return a
+}
+
+//Sent gives packets sent
```

Comment:
Also the comment is ambiguous: it "gives the packets sent" sounds more like it returns the raw TCP packages, there is wireshark for that ;-)